django/contrib/gis/geos/__init__.py

```diff
@@ -9,7 +9,7 @@
 from .factory import fromfile, fromstr  # NOQA
 from .geometry import GEOSGeometry, hex_regex, wkt_regex  # NOQA
 from .io import WKBReader, WKBWriter, WKTReader, WKTWriter  # NOQA
-from .libgeos import geos_version, geos_version_info  # NOQA
+from .libgeos import geos_version, geos_version_tuple  # NOQA
```

Comment:
Not sure if this change is needed -- doesn't seem like Django is importing from here?